bot.js

```diff
@@ -629,11 +631,11 @@ function pretty_char2num(ch) {
     if (ch === ".") return -1;
     return "abcdefghjklmnopqrstuvwxyz".indexOf(ch.toLowerCase());
 }
-function move2gtpvertex(move, board_size) {
+function move2gtpvertex(move, bot, height) {
```

Comment:
here the `bot` parameter seems unused
something missing ?

Reply:
ah this is 'width', and is not used but here so the functions all look the same